worldprops/default.conf

```diff
@@ -0,0 +1,71 @@
+[server]
+#Secret key - this is sent by the client during login.
+#Client/Server MUST match or connection is refused.
+secret_key = "2009scape_development"
+write_logs = true
+msip = "127.0.0.1"
+
+[database]
+database_name = "global"
+database_username = "root"
+database_password = ""
+database_address = "127.0.0.1"
+database_port = "3306"
+
+
+[world]
+name = "2009scape"
+debug = false
+dev = false
+start_gui = false
+daily_restart = true
+#world number
+world_id = "1"
+country_id = "0"
+members = true
+#activity as displayed on the world list
+activity = "2009scape classic."
+pvp = false
+default_xp_rate = 25.0
```

Comment:
ah, forgot to change these back -- will revert my local changes to this file back to what was on master